### PR TITLE
[chore] update codeowners for security

### PR DESCRIPTION
**Pull request by trisch-me**
According to the https://github.com/open-telemetry/community/issues/2100 the approvers group was created

I have update codeowners file to reflect changes

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -76,4 +76,13 @@
 /model/trace/gen-ai.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-llm-approvers
 /docs/gen-ai/ @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-llm-approvers
 
+# Security semantic conventions approvers
+/model/registry/file.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/dns.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/network.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/process.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/tls.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/user.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
```
Comment (trask) [inline .github/CODEOWNERS:85]: do you want to include these?

* `/model/metrics/dns.yaml`
* `/model/network.yaml`
* `/model/registry/deprecated/network.yaml`
* `/model/resource/process.yaml`

Comment (trisch-me) [reply .github/CODEOWNERS:85]: thanks @trask, sure. I somehow forgot we define things not only in registry :)

**Review by trask — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -76,4 +76,17 @@
 /model/trace/gen-ai.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-llm-approvers
 /docs/gen-ai/ @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-llm-approvers
 
+# Security semantic conventions approvers
+/model/registry/file.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/dns.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/network.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
```
Comment (lmolkova) [inline .github/CODEOWNERS:82]: it's already defined on line 40 (and owners include semconv-http-approvers)

Comment (trisch-me) [reply .github/CODEOWNERS]: fixed, thanks for noticing it

**Review by lmolkova — APPROVED** [review]

**Pull request merged**